### ISSUE TITLE
domoticz does not start with the Domoticz-deCONZ plugin

### ISSUE THREAD

**Issue by ryanrudak** [Smanar/Domoticz-deCONZ#45]
Linux 4.9.0-11-amd64 #1 SMP Debian 4.9.189-3+deb9u2 (2019-11-11) x86_64 GNU/Linux
debian stretch
Domoticz V4.11690
requests 2.12.4
pyopenssl 2.16.0

When I comment 'import requests' in the plugin.py out, then domoticz starts.

Output of domoticz.log

2020-02-14 20:18:04.681  Error: Domoticz(pid:12416, tid:12419('PluginMgr')) received fatal signal 11 (Segmentation fault)
2020-02-14 20:18:04.682  Error: siginfo address=0x3080, address=0x7f35d92e9fbf
2020-02-14 20:18:04.707  Error: Failed to start gdb, will use backtrace() for printing stack frame

2020-02-14 20:18:04.715  Error: #0  /home/domo/domoticz/domoticz() [0x735e53]
2020-02-14 20:18:04.716  Error: #1  /home/domo/domoticz/domoticz : signal_handler(int, siginfo_t*, void*) + 0x295 [0x736975]
2020-02-14 20:18:04.716  Error: #2  /lib/x86_64-linux-gnu/libpthread.so.0 :  + 0x110e0 [0x7f35d92ea0e0]
2020-02-14 20:18:04.716  Error: #3  /lib/x86_64-linux-gnu/libpthread.so.0 : raise + 0xcf [0x7f35d92e9fbf]
2020-02-14 20:18:04.717  Error: #4  /lib/x86_64-linux-gnu/libpthread.so.0 :  + 0x110e0 [0x7f35d92ea0e0]
2020-02-14 20:18:04.717  Error: #5  /lib/x86_64-linux-gnu/libc.so.6 :  + 0x8f53a [0x7f35d86a253a]
2020-02-14 20:18:04.717  Error: #6  /home/domo/domoticz/domoticz() [0xc47429]
2020-02-14 20:18:04.717  Error: #7  /home/domo/domoticz/domoticz : lh_insert + 0x4c [0xc476dc]
2020-02-14 20:18:04.718  Error: #8  /home/domo/domoticz/domoticz : OBJ_NAME_add + 0x63 [0xc10543]
2020-02-14 20:18:04.718  Error: #9  /home/domo/domoticz/domoticz : EVP_add_digest + 0x56 [0xc4ef76]
2020-02-14 20:18:04.718  Error: #10 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 :  + 0x1554fb [0x7f35b84b44fb]
2020-02-14 20:18:04.719  Error: #11 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 :  + 0x16ca79 [0x7f35b84cba79]
2020-02-14 20:18:04.719  Error: #12 /lib/x86_64-linux-gnu/libpthread.so.0 :  + 0xe759 [0x7f35d92e7759]
2020-02-14 20:18:04.719  Error: #13 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 : CRYPTO_THREAD_run_once + 0x9 [0x7f35b8522759]
2020-02-14 20:18:04.720  Error: #14 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 : OPENSSL_init_crypto + 0x273 [0x7f35b84cbfb3]
2020-02-14 20:18:04.720  Error: #15 /usr/lib/python3.5/lib-dynload/_hashlib.cpython-35m-x86_64-linux-gnu.so : PyInit__hashlib + 0x21 [0x7f35c85f6b11]
2020-02-14 20:18:04.720  Error: #16 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : _PyImport_LoadDynamicModuleWithSpec + 0x180 [0x7f35d478b710]
2020-02-14 20:18:04.720  Error: #17 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x209fe7 [0x7f35d478efe7]
2020-02-14 20:18:04.721  Error: #18 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xc9 [0x7f35d45f6259]
2020-02-14 20:18:04.721  Error: #19 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x78f2 [0x7f35d4726ff2]
2020-02-14 20:18:04.721  Error: #20 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.722  Error: #21 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x63df [0x7f35d4725adf]
2020-02-14 20:18:04.722  Error: #22 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.722  Error: #23 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.723  Error: #24 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.723  Error: #25 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.723  Error: #26 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.723  Error: #27 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.724  Error: #28 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0xf7558 [0x7f35d467c558]
2020-02-14 20:18:04.724  Error: #29 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.724  Error: #30 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : _PyObject_CallMethodIdObjArgs + 0x22b [0x7f35d475ae1b]
2020-02-14 20:18:04.724  Error: #31 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyImport_ImportModuleLevelObject + 0x81a [0x7f35d478cffa]
2020-02-14 20:18:04.725  Error: #32 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a6248 [0x7f35d472b248]
2020-02-14 20:18:04.725  Error: #33 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xe9 [0x7f35d45f6279]
2020-02-14 20:18:04.725  Error: #34 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.725  Error: #35 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_CallObjectWithKeywords + 0x47 [0x7f35d47e7b77]
2020-02-14 20:18:04.726  Error: #36 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x40cb [0x7f35d47237cb]
2020-02-14 20:18:04.726  Error: #37 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.726  Error: #38 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.726  Error: #39 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCode + 0x1b [0x7f35d471f45b]
2020-02-14 20:18:04.726  Error: #40 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a72cd [0x7f35d472c2cd]
2020-02-14 20:18:04.726  Error: #41 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xc9 [0x7f35d45f6259]
2020-02-14 20:18:04.727  Error: #42 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x78f2 [0x7f35d4726ff2]
2020-02-14 20:18:04.727  Error: #43 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.727  Error: #44 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x63df [0x7f35d4725adf]
2020-02-14 20:18:04.727  Error: #45 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.727  Error: #46 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.728  Error: #47 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.728  Error: #48 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.728  Error: #49 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.728  Error: #50 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0xf7558 [0x7f35d467c558]
2020-02-14 20:18:04.728  Error: #51 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.729  Error: #52 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : _PyObject_CallMethodIdObjArgs + 0x22b [0x7f35d475ae1b]
2020-02-14 20:18:04.729  Error: #53 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyImport_ImportModuleLevelObject + 0x81a [0x7f35d478cffa]
2020-02-14 20:18:04.729  Error: #54 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a6248 [0x7f35d472b248]
2020-02-14 20:18:04.729  Error: #55 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xe9 [0x7f35d45f6279]
2020-02-14 20:18:04.729  Error: #56 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.730  Error: #57 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_CallObjectWithKeywords + 0x47 [0x7f35d47e7b77]
2020-02-14 20:18:04.730  Error: #58 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x40cb [0x7f35d47237cb]
2020-02-14 20:18:04.730  Error: #59 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.730  Error: #60 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.730  Error: #61 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCode + 0x1b [0x7f35d471f45b]
2020-02-14 20:18:04.730  Error: #62 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a72cd [0x7f35d472c2cd]
2020-02-14 20:18:04.731  Error: #63 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xc9 [0x7f35d45f6259]
2020-02-14 20:18:04.731  Error: #64 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x78f2 [0x7f35d4726ff2]
2020-02-14 20:18:04.731  Error: #65 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.731  Error: #66 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x63df [0x7f35d4725adf]
2020-02-14 20:18:04.731  Error: #67 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.731  Error: #68 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.731  Error: #69 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.731  Error: #70 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.731  Error: #71 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.731  Error: #72 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0xf7558 [0x7f35d467c558]
2020-02-14 20:18:04.731  Error: #73 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.732  Error: #74 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : _PyObject_CallMethodIdObjArgs + 0x22b [0x7f35d475ae1b]
2020-02-14 20:18:04.732  Error: #75 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyImport_ImportModuleLevelObject + 0x81a [0x7f35d478cffa]
2020-02-14 20:18:04.732  Error: #76 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a6248 [0x7f35d472b248]
2020-02-14 20:18:04.732  Error: #77 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xe9 [0x7f35d45f6279]
2020-02-14 20:18:04.732  Error: #78 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.732  Error: #79 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_CallObjectWithKeywords + 0x47 [0x7f35d47e7b77]
2020-02-14 20:18:04.732  Error: #80 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x40cb [0x7f35d47237cb]
2020-02-14 20:18:04.732  Error: #81 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.732  Error: #82 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.732  Error: #83 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCode + 0x1b [0x7f35d471f45b]
2020-02-14 20:18:04.732  Error: #84 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a72cd [0x7f35d472c2cd]
2020-02-14 20:18:04.732  Error: #85 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xc9 [0x7f35d45f6259]
2020-02-14 20:18:04.732  Error: #86 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x78f2 [0x7f35d4726ff2]
2020-02-14 20:18:04.732  Error: #87 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.732  Error: #88 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x63df [0x7f35d4725adf]
2020-02-14 20:18:04.732  Error: #89 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.732  Error: #90 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.732  Error: #91 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.733  Error: #92 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.733  Error: #93 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.734  Error: #94 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0xf7558 [0x7f35d467c558]
2020-02-14 20:18:04.734  Error: #95 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.734  Error: #96 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : _PyObject_CallMethodIdObjArgs + 0x22b [0x7f35d475ae1b]
2020-02-14 20:18:04.735  Error: #97 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyImport_ImportModuleLevelObject + 0x81a [0x7f35d478cffa]
2020-02-14 20:18:04.735  Error: #98 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a6248 [0x7f35d472b248]
2020-02-14 20:18:04.735  Error: #99 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xe9 [0x7f35d45f6279]
2020-02-14 20:18:04.735  Error: #100 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x78f2 [0x7f35d4726ff2]
2020-02-14 20:18:04.736  Error: #101 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.736  Error: #102 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x63df [0x7f35d4725adf]
2020-02-14 20:18:04.736  Error: #103 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.737  Error: #104 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.737  Error: #105 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0xf7558 [0x7f35d467c558]
2020-02-14 20:18:04.737  Error: #106 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.738  Error: #107 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : _PyObject_CallMethodIdObjArgs + 0x22b [0x7f35d475ae1b]
2020-02-14 20:18:04.738  Error: #108 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyImport_ImportModuleLevelObject + 0x645 [0x7f35d478ce25]
2020-02-14 20:18:04.738  Error: #109 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a6248 [0x7f35d472b248]
2020-02-14 20:18:04.738  Error: #110 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xe9 [0x7f35d45f6279]
2020-02-14 20:18:04.739  Error: #111 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyObject_Call + 0x47 [0x7f35d4758a37]
2020-02-14 20:18:04.739  Error: #112 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_CallObjectWithKeywords + 0x47 [0x7f35d47e7b77]
2020-02-14 20:18:04.739  Error: #113 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x40cb [0x7f35d47237cb]
2020-02-14 20:18:04.739  Error: #114 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.740  Error: #115 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.740  Error: #116 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCode + 0x1b [0x7f35d471f45b]
2020-02-14 20:18:04.740  Error: #117 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x1a72cd [0x7f35d472c2cd]
2020-02-14 20:18:04.740  Error: #118 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyCFunction_Call + 0xc9 [0x7f35d45f6259]
2020-02-14 20:18:04.741  Error: #119 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x78f2 [0x7f35d4726ff2]
2020-02-14 20:18:04.741  Error: #120 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.741  Error: #121 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x63df [0x7f35d4725adf]
2020-02-14 20:18:04.742  Error: #122 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.742  Error: #123 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.742  Error: #124 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalFrameEx + 0x7fad [0x7f35d47276ad]
2020-02-14 20:18:04.742  Error: #125 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0x264074 [0x7f35d47e9074]
2020-02-14 20:18:04.742  Error: #126 /usr/lib/x86_64-linux-gnu/libpython3.5m.so : PyEval_EvalCodeEx + 0x23 [0x7f35d47e9153]
2020-02-14 20:18:04.742  Error: #127 /usr/lib/x86_64-linux-gnu/libpython3.5m.so :  + 0xf7558 [0x7f35d467c558]

**Comment by Smanar** [comment]
Ok so bad luck.
There is someone with the same problem than you https://www.domoticz.com/forum/viewtopic.php?f=68&t=25631&start=320#p227552

And the only solution he have find, is a complete re-installatioin from scratch.

And there is a thing I can't understand, you are using the last domoticz beta, and you are still on stretch ? From the forum the last beta works only on buster ?

Another person with the last beta https://www.domoticz.com/forum/viewtopic.php?f=65&t=21645&p=236837&hilit=fatal+signal+11#p236837

Have you take a look on the post about new beta and buster ?

**Comment by ryanrudak** [comment]
Hi Smanar,

thanks for your fast response.
No, have not seen the post about the new beta and debian 10 (buster). the new beta was ok on my system until I tryed the plugin.

Now I have fixed the problem.
(In my case, a re-installation from scratch should be the last option ;-) )

What I did:
first I upgraded from debian 9 (stretch) to debian 10 (buster) - was easy and runs without errors
--> but the error was the same

second I tryed to re-install a lot of pip/pip3 modules and some of the debian packages like python, libpython, openssl, cryptography and so on...
--> but the error was the same and did not help

third I tryed to downgrade to the stable domoticz version
--> did not help because the database version (from the last beta) was not compatible anymore

fourth I read in a post (a lot of posts I read) to re-compile domoticz
(https://www.domoticz.com/wiki/Build_Domoticz_from_source)
--> THAT helps :-) and now your Domoticz-deCONZ plugin runs

In my case I re-compiled like described in the domoticz article and copied the fresh compiled domoticz file (only that one file) to my origin domoticz installation.

Thanks for your nice work!
